realm/src/androidTest/java/io/realm/RealmTest.java

```diff
@@ -790,7 +791,26 @@ public void rarely_run_testUTF8() {
             }
             testRealm.commitTransaction();
         } catch (Exception e) {
-            fail("Failure, Codepoint: " + i + " / " + currentUnicode  + " " +  e.getMessage());
+            fail("Failure, Codepoint: " + i + " / " + currentUnicode + " " + e.getMessage());
+        }
+    }
+
+    //tests for closing open realms
+    public void testOutOfFile() {
```

Comment:
What does this test, that couldn't be accomplished by just doing it once. Needs either to be documented or refactored.